Referenced by issue #94708, New-file causes Python assert with negative reference count
